.circleci/config.yml

```diff
@@ -13,22 +13,29 @@ commands:
       - run: brew install protobuf cmake ccache libtool boost-python libspatialite pkg-config luajit curl wget czmq lz4 spatialite-tools unzip
 
 jobs:
-  build-base:
+  build-docker-images:
     docker:
       - image: docker:18.06.3-ce-git
     steps:
       - setup_remote_docker
       - checkout
       - run:
-          name: Build Valhalla base Dockerfile
+          name: Build Valhalla Docker Images
           command: |
             docker build -f ./docker/Dockerfile-build --tag valhalla/valhalla:build-latest ./docker
+            if [[ "${CIRCLE_BRANCH}" == "master" ]] || [[ ! -z "${CIRCLE_TAG}" ]]; then
```

Comment:
if we are merging master or tagging then we also building the binary artifacts image